index.html

```diff
@@ -58,16 +138,46 @@ <h1 class="logo">
     <section class="About-section">
       <div class="about">About</div>
       <p class="about-info">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
-        eiusmod
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
       </p>
-      <button type="button" class="cv">
+      <button type="button" class="connect">
         <img src="Group86.png" alt="yes" />
       </button>
       <div></div>
     </section>
-    <section></section>
+    <section class="end">
+      <div class="skill-containor">
+        <div class="tileone">Languages</div>
```

Comment:
- [ ] Kindly change all `<div class="tileone">` to one of the `hx(h2-h6)` tags to make it more semantic.